### DISCUSSION TITLE
LocalPath - The path used to clone the repo to for updating. Defaults to $(System.DefaultWorkingDirectory)\repo

### DISCUSSION THREAD

**Discussion by Cestmoileo** [rfennell/AzurePipelines#1520] [Q&A]
LocalPath - The path used to clone the repo to for updating. Defaults to $(System.DefaultWorkingDirectory)\repo

Hello,

I need help , i am using both of your tasks for the generation of release notes in my azure devops portal ,  please let me know whether the below said parameter is mandatory because it creates a sub page in my git wiki which i dont want to get cloned there.
Thats not great looking, is there any other workaround for this , please help !!

LocalPath - The path used to clone the repo to for updating. Defaults to $(System.DefaultWorkingDirectory)\repo

**Comment by Cestmoileo** [comment]
[](url)
![releasenotes](https://github.com/rfennell/AzurePipelines/assets/39943346/f1de8c42-d289-44c1-b341-82511e920d21)

**Reply by rfennell** [reply]
The `LocalPath - The path used to clone the repo to for updating. Defaults to $(System.DefaultWorkingDirectory)\repo` is just the temporary folder used for the local cloned copy of the WIKI repo.

The file to add to the WIKI is copied into the correct folder  then a commit done. 

So the only thing that should get uploaded to the WIKI is the file/folder stucture added to this temp repo

Does that help?

**Reply by Cestmoileo** [reply]
Thanks for replying back 👍 , yes correct. The folder structure gets copied over there in my wiki location in the folder named "repo".
But i dont want the folder and its files to be copied again over there and visible in my wiki location. is there something you could help me with ? Its like a duplicate/ cloned copy of the same folder structure under "repo"

![image](https://github.com/rfennell/AzurePipelines/assets/39943346/72b95844-df03-472d-9ac4-85bb817e3b3d)

Thanks for this excellent  and awesome tool !! 

Léo
